pkg/chartutil/values_test.go

```diff
@@ -316,6 +322,8 @@ func TestCoalesceValues(t *testing.T) {
 		expect string
 	}{
 		{"{{.top}}", "yup"},
+		// The keys bottom, right, left, front, and back should not be present.
+		{"{{.side}}", ""},
```

Comment:
Definitely a good point. We are talking about this now in Slack to get some ideas